client/app/telecom/telephony/billingAccount/administration/linesGroup/telecom-telephony-billing-account-administration-lines-group.controller.js

```diff
@@ -80,11 +80,29 @@ angular.module('managerApp').controller('TelecomTelephonyBillingAccountAdministr
         ))
         .then(chunkResult => _.pluck(_.flatten(chunkResult), 'value')));
 
+    // get batch voicefax details
+    const voicefax = OvhApiTelephony.Fax().v6()
+      .query({
+        billingAccount: ba.billingAccount,
+      }).$promise
+      .then(ids => $q
+        .all(_.map(
+          _.chunk(ids, 50),
+          chunkIds => OvhApiTelephony.Fax().v6().getBatch({
+            billingAccount: ba.billingAccount,
+            serviceName: chunkIds,
```

Comment:
Don't you need to `join(',')` ?

Reply:
No, I did the same than with the call of OvhApiTelephony.Line().v6().getBatch(...)